packages/office-ui-fabric-react/src/components/Text/examples/Text.Ramp.Example.tsx

```diff
@@ -0,0 +1,91 @@
+// @codepen
+import { Text } from 'office-ui-fabric-react/lib/Text';
+import { IFontStyles } from 'office-ui-fabric-react/lib/Styling';
+import {
+  DetailsList,
+  IColumn,
+  DetailsListLayoutMode,
+  SelectionMode,
+  DetailsRow,
+  IDetailsRowProps
+} from 'office-ui-fabric-react/lib/DetailsList';
+import * as React from 'react';
+
+const TestText = 'The quick brown fox jumped over the lazy dog.';
+
+interface ISetting<TType> {
+  name: TType;
+}
+
+const Variants: ISetting<keyof IFontStyles>[] = [
+  { name: 'tiny' },
+  { name: 'xSmall' },
+  { name: 'small' },
+  { name: 'smallPlus' },
+  { name: 'medium' },
+  { name: 'mediumPlus' },
+  { name: 'large' },
+  { name: 'xLarge' },
+  { name: 'xxLarge' },
+  { name: 'mega' }
+];
+
+export interface ITextRampExampleItem {
+  key: string;
+  variant: string;
+  example: JSX.Element;
+}
+
+export interface ITextRampExampleState {
+  items: ITextRampExampleItem[];
+}
+
+export class TextRampExample extends React.Component<{}, ITextRampExampleState> {
+  private _allItems: ITextRampExampleItem[];
+  private _columns: IColumn[];
+
+  constructor(props: {}) {
+    super(props);
+
+    this._allItems = [];
+    Variants.map((setting: ISetting<keyof IFontStyles>, index: number) =>
```

Comment:
Consider returning a value from the `map` callback and assigning the map call to `this._allItems`. `Array.prototype.map` allocates an array based on what is returned from the callback.

>The map() method creates a new array with the results of calling a provided function on every element in the calling array.
>https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map

Otherwise, I suggest modifying this to a `forEach`.